.travis.yml

```diff
@@ -1,34 +1,31 @@
 ---
 language: node_js
 node_js:
+  # we recommend testing addons with the same minimum supported node version as Ember CLI
+  # so that your addon works for all apps
   - "4"
 
+sudo: false
 dist: trusty
-sudo: required
 
 addons:
-  apt:
-    sources:
-    - google-chrome
-    packages:
-    - google-chrome-stable
+  chrome: stable
 
 cache:
   yarn: true
 
-before_install:
-  - "export DISPLAY=:99.0"
-  - "sh -e /etc/init.d/xvfb start"
-  - sleep 3
-  - curl -o- -L https://yarnpkg.com/install.sh | bash
-  - export PATH=$HOME/.yarn/bin:$PATH
-  - yarn global add bower
+env:
+  global:
+    # See https://git.io/vdao3 for details.
+    - JOBS=1
+
+matrix:
+  fast_finish: true
 
-install:
-  - yarn install --no-lockfile
-  - bower install
+before_install:
+  - npm config set spin false
+  - npm install -g npm@4
+  - npm --version
```

Comment:
I'd prefer to stick with yarn

Reply:
Ooops! Got a little confused syncing up the diff between the blueprint and here.  I think I got it back to full yarn usage now.  Let me know if I missed something.